From foo@baz Mon Nov 24 21:15:22 PST 2008
Date: Mon, 24 Nov 2008 21:15:22 -0800
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: benet: fix build error.

udelay() was not being defined properly, this patch fixes it.

Reported-by: Randy Dunlap <randy.dunlap@oracle.com>
Cc: Subbu Seetharaman <subbus@serverengines.com>

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/benet/mpu.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/benet/mpu.c
+++ b/drivers/staging/benet/mpu.c
@@ -14,6 +14,7 @@
  * 209 N. Fair Oaks Ave
  * Sunnyvale, CA 94085
  */
+#include <linux/delay.h>
 #include "hwlib.h"
 #include "bestatus.h"
 
